contracts/debt/IncurDebtV1.sol

```diff
@@ -0,0 +1,453 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import "../interfaces/IOHM.sol";
+import "../interfaces/IsOHM.sol";
+import "../interfaces/IgOHM.sol";
+import "../interfaces/IERC20.sol";
+import "../interfaces/IStaking.sol";
+import "./interfaces/ITreasury.sol";
+import "../libraries/SafeERC20.sol";
+import "../types/OlympusAccessControlledV2.sol";
+
+error IncurDebtV1_NotBorrower(address _borrower);
+error IncurDebtV1_InvaildNumber(uint256 _amount);
+error IncurDebtV1_WrongTokenAddress(address _token);
+error IncurDebtV1_AlreadyBorrower(address _borrower);
+error IncurDebtV1_AboveGlobalDebtLimit(uint256 _limit);
+error IncurDebtV1_AboveBorrowersDebtLimit(uint256 _limit);
+error IncurDebtV1_LimitBelowOutstandingDebt(uint256 _limit);
+error IncurDebtV1_AmountAboveBorrowerBalance(uint256 _amount);
+error IncurDebtV1_AmountMoreThanBorrowersLimit(uint256 _borrower);
+error IncurDebtV1_OHMAmountMoreThanAvailableLoan(uint256 _amount);
```

Comment:
isnt this basically same as IncurDebtV1_AmountAboveBorrowerBalance ?

Reply:
Yup same, will remove.

Reply:
yea but its still referring to the same numbers right. The ohm amount the user input is > getAvailableToBorrow(). Small thing to nitpick on but i think it would be bit neater if we use the same error message for errors that are similar in nature.

Reply:
The essence of having error messages is to help easily track where errors are coming from, using a general term makes it difficult and is time wasting for any dev interacting with the supposed contract in terms of pinpointing where an error comes from.